### DISCUSSION TITLE
File support: vector indexing & retrieval

### DISCUSSION THREAD

**Discussion by thunderbug1** [danny-avila/LibreChat#509] [General]
@danny-avila, do you need backend help with this item? "File support: vector indexing & retrieval"

What is the concept to implement this? 
As a plugin or directly integrated in the UI?

**Comment by danny-avila** [comment]
> @danny-avila, do you need backend help with this item? "File support: vector indexing & retrieval"


Any help is appreciated. Best is to reach me on discord as I can miss posts here.

> What is the concept to implement this? As a plugin or directly integrated in the UI?


Both, let's discuss!
